hugegraph-api/src/main/java/org/apache/hugegraph/api/filter/AccessLogFilter.java

```diff
@@ -62,13 +79,24 @@ public void filter(ContainerRequestContext requestContext, ContainerResponseCont
 
         // get responseTime
         Object requestTime = requestContext.getProperty(REQUEST_TIME);
-        if(requestTime!=null){
+        if(requestTime != null){
             long now = System.currentTimeMillis();
-            long responseTime = (now - (long)requestTime);
+            long start = (Long) requestTime;
+            long responseTime = now - start;
 
             MetricsUtil.registerHistogram(
                                join(metricsName, METRICS_PATH_RESPONSE_TIME_HISTOGRAM))
                        .update(responseTime);
+
+            HugeConfig config = configProvider.get();
+            Long timeThreshold = config.get(ServerOptions.SLOW_QUERY_LOG_TIME_THRESHOLD);
```

Comment:
Holding the timeThreshold in a private variable and changing its type to long would be better, reducing the cost of unboxing and hash searching.

Reply:
thanks ， but i dont know “Holding the timeThreshold in a private variable ” means what？ thanks again

Reply:
Just like this:
```
public class AccessLogFilter implements ContainerResponseFilter {
     private long timeThreshold =configProvider.get().get(ServerOptions.SLOW_QUERY_LOG_TIME_THRESHOLD);

.....
}
```
Of course, you need to check for null.

Reply:
> Just like this:
> 
> ```
> public class AccessLogFilter implements ContainerResponseFilter {
>      private long timeThreshold =configProvider.get().get(ServerOptions.SLOW_QUERY_LOG_TIME_THRESHOLD);
> 
> .....
> }
> ```
> 
> Of course, you need to check for null.

we cant get timeThreshold as private var from configProvider, because configProvider managed by @Context; it will case NPE.